integration-tests/jakarta-ee/src/main/java/org/apache/shiro/testing/jakarta/ee/FormBean.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.shiro.testing.jakarta.ee;
+
+import javax.enterprise.inject.Model;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.shiro.ee.filters.Forms;
+import org.omnifaces.util.Faces;
+import org.omnifaces.util.Messages;
+
+/**
+ *
+ * @author lprimak
```

Comment:
Do we need `@author` ? Looking at Shiro's history, I found SHIRO-168, which bulk removed the tags via

```bash
SHIRO-168 - remove all @author tags using:
find . | xargs grep -l "@author" | grep -v "\.svn" | xargs sed -i.bak "/@author/d"
```